protocol/signalfx/format/log/gen_go/signalfx_log.pb.go

```diff
@@ -0,0 +1,712 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
You may also want to explore Gogoproto: https://github.com/gogo/protobuf. It is compatible with goproto but generates faster code.